Systems ref jcayouette (Content Lifecycle Update) (#433)
SUSE/doc-susemanager: modules/reference/pages/configuration/config-systems-managed.adoc

[[config-systems-managed]]
= Managed Systems

By default the [guimenu]``Managed Systems`` page is displayed.
The listed systems have been fully prepared for configuration file deployment.

> The number of locally- and centrally-managed files is displayed.

Reviewer: ```suggestion
The number of locally-managed and centrally-managed files is displayed.
```

> Clicking the name of a system shows its menu:System Details[Configuration > Overview] page.

Reviewer: ```suggestion
Click the name of a system to show the relevant menu:System Details[Configuration > Overview] page.
```

> Clicking the number of local files takes you to the menu:System Details[Configuration > View/Modify Files > Locally-Managed Files] page, where you manage which local (override) files apply to the system.

Reviewer: ```suggestion
Click the number of local files to show the menu:System Details[Configuration > View/Modify Files > Locally-Managed Files] page, where you can manage which local (override) files apply to the system.
```

> Clicking the number of centrally-managed files takes you to the menu:System Details[Configuration > Manage Configuration Channels > List/Unsubscribe    from Channels] page.

Reviewer: ```suggestion
Click the number of centrally-managed files to show the menu:System Details[Configuration > Manage Configuration Channels > List/Unsubscribe from Channels] page.
```

> Here you unsubscribe from any channels you want.

Reviewer: ```suggestion
On this page, you can unsubscribe from channels.
```
